src/test-utils/dom/steps/index.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentWrapper, ElementWrapper } from '@cloudscape-design/test-utils-core/dom';
+
+import statusIconStyles from '../../../status-indicator/styles.selectors.js';
+import testUtilStyles from '../../../steps/test-classes/styles.selectors.js';
+
+export class StepWrapper extends ComponentWrapper {
+  /**
+   * Finds the status of a step
+   */
+  findStatusIndicator(): ElementWrapper | null {
+    return this.findByClassName(statusIconStyles.icon);
+  }
+
+  /**
+   * Finds the header of a step
+   */
+  findHeader(): ElementWrapper | null {
+    return this.findByClassName(testUtilStyles['step-header']);
+  }
+
+  /**
+   * Finds the details of a step
+   */
+  findDetails(): ElementWrapper | null {
+    return this.findByClassName(testUtilStyles['step-details']);
+  }
+}
+export default class StepsWrapper extends ComponentWrapper {
+  static rootSelector: string = testUtilStyles.steps;
```

Comment:
The same case with my comment above, you don't need the testUtilStyles, here you can use `styles.root` which is usually how it's defined for the other components as well.